### ISSUE TITLE
BTd slots are not reactive

### ISSUE THREAD

**Issue by danielelkington** [cdmoro/bootstrap-vue-3#345]
Consider a `b-table-simple` with some dynamic content:
```html
<b-table-simple>
<b-tbody>
  <b-tr>
    <b-td>{{count}}</b-td>
    <b-td><button @click="count++">Increment</button></b-td>
  </b-tr>
</b-tbody>
```
```js
// in script
const count = ref(count)
```
I would expect that, as-per `bootstrap-vue`, the slot would be dynamic - that is, clicking the button updates the count. However it seems that the slot content is only rendered once and is not reactive.

## Workaround
The workaround would be to force the BTd to re-render when the reactive data changes, such as by using a `key` attribute.
```html
<b-td :key="count">{{count}}</b-td>
```

**Comment by cdmoro** [comment]
I found where the error is! I'll fix it in a little while :smiley: thanks!

**Comment by cdmoro** [comment]
Closed by #347

**Comment by danielelkington** [comment]
Thanks; looks great!
